docs/topics/install/docker.rst

```diff
@@ -10,83 +10,78 @@ development environment.
 First you'll need to install docker_, please check the information for
 the installation steps specific to your operating system.
 
-.. note::
-    Docker recommends installing docker-toolbox_ if you're on OS X or
-    windows and that will provide you with the ``docker-machine`` and
-    ``docker-compose`` (Mac-only).
-
+There are generally two options for running docker depending on the platform
+you are running.
 
-.. _creating-the-docker-vm:
+ * Run docker on the host machine directly (recommended if supported)
+ * Run docker-machine which will run docker inside a virtual-machine
 
-Creating the docker vm (mac/windows)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Historically mac and windows could only run docker via a vm. That has
+recently changed with the arrival of docker-for-mac_ and docker-for-windows_.
 
-If you go the docker-machine route your first step is to create a vm::
+If your platform can run docker directly either on Linux, with docker-for-mac_
+or docker-for-windows_ then this is the easiest way to install docker with the
+most minimal set of moving parts.
 
-    docker-machine create --driver=virtualbox addons-dev
+If you have problems, due to not meeting the requirenents or you're on windows
+and want to keep running docker-machine vms then docker-machine will still
+work just fine. See the docs for creating the vm here :ref:`creating-the-docker-vm`
 
-Then you can export the variables so that docker-compose can talk to
-the docker service. This command will tell you how to do that::
+.. note:: if you're on OSX and already have a working docker-machine setup you
```

Comment:
While it's technically "OS X", I'd change this to "Mac" or "MacOS" as they are getting away from the OS X branding anyway and "Mac" stands out more.